.gitignore

```diff
@@ -7,3 +7,4 @@ build
 project.xcworkspace
 xcuserdata
 Credits.rtf
+C_count.occ.exe
```

Comment:
Could you comment here what this `.exe` is for?

Reply:
It was a deprecated utility, I will comment it and remove the dead stuff in the next PR.